### ISSUE TITLE
Make throttling optional

### ISSUE THREAD

**Issue by denar90** [denar90/sloth#1]
As a user, I want to have dropdown to choose/type throttling values

**Comment by denar90** [comment]
Expected options:
 - fast mobile (includes CPU and 4G network throttling)
 - slow mobile (includes CPU and 3G network throttling)
 - fast desktop (4G network throttling)
 - slow desktop (3G network throttling)

**Comment by abhisheksoni27** [comment]
Doesn't that complicate things further? As the readme states, if a user is willing to open the devtools and choose whatever options they like, what is the need for sloth anyway?

And the answer sums it up. It is for lazy people who do not want to open DevTools and click through all the hurdles just to be able to enable throttling.

If you were to add those options here, then it might reduce one step compared to the DevTools route but it totally beats the purpose of the extension.

**Comment by denar90** [comment]
Yes you re right. But we store user selection in a profile, so it will be almost one time move. Motivation behind this - throttling used in Sloth might be to rough for users with too slow working machines. So they can select what better for them to have: throttling and proper level slowness to have deal with browsing the site. 

During wring the reply I realized that developer can make it faster (4G desktop) and suppose it's good and forget about slow devices.

@abhisheksoni27 thanks, I agree, let's keep as is. In case options are needed we can reconsider adding them.

**Comment by Suleman-Elahi** [comment]
> Expected options:
> 
> * fast mobile (includes CPU and 4G network throttling)
> * slow mobile (includes CPU and 3G network throttling)
> * fast desktop (4G network throttling)
> * slow desktop (3G network throttling)

No need for these but an option for adding custom download, upload, and latency would be great !!!

**Comment by denar90** [comment]
Since we have settings window in new design

![image](https://user-images.githubusercontent.com/6231516/57106594-7af44080-6d36-11e9-8f33-16df0e4db206.png)

there might be place for that.

Also will be nice to stick to Lighthouse approach, like that one

![image](https://user-images.githubusercontent.com/6231516/57106716-cb6b9e00-6d36-11e9-9a2b-bbe239de7633.png)

Is it something interesting you'd like to work on @Suleman-Elahi ?

**Comment by Suleman-Elahi** [comment]
I'd love to but I never tried my hands on JS or Chrome Extensions :[]

**Comment by denar90** [comment]
I can assist in case you'd like to try :)
